### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by Freedle** [lucatauer/dmarket-buff-percentage-chrome#1]
Hi. 
When running your extension, an error appears.
![image](https://user-images.githubusercontent.com/10924611/226219865-5718a894-f4e3-419e-8c12-685f22931f84.png)

**Comment by lucatauer** [comment]
Hey, thanks for the message! I'm working on an update to Manifest version 3, but the extension should work for now, does it work for you?

**Comment by Freedle** [comment]
It's funny, but I checked again today and it worked. But for most skins, the percentage is not displayed.

**Comment by lucatauer** [comment]
Yes, I know, I've only had time for some of the main skins so far, but I'll add the rest next week at the latest, sorry!

**Comment by lucatauer** [comment]
Thanks!
